gpkit/constraints/geometric_program.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 from ..nomials import NomialData
 from ..small_classes import CootMatrix, SolverLog
-from ..keydict import FastKeyDict
```

Comment:
I was thinking we'd keep it for now as an example of a definitely unmapped dict, but there's no real reason to, and it's fun to remove code, so I'll do that. :D